<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PC_Constructor</class>
 <widget class="QMainWindow" name="PC_Constructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PC_Constructor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>Колекція </string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>ПК1</string>
         </property>
         <item>
          <property name="text">
           <string>Компоненти</string>
          </property>
          <item>
           <property name="text">
            <string>Процесор</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Блок живлення</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Характеристики</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>ПК2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>ПК1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>ПК1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Компоненти</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Components" name="widget" native="true"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Характеристики</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Specifications" name="widget_2" native="true"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Зберегти</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Копіювати</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Видалити</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>ПК1 &gt; Компоненти</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>ПК1 &gt; Характеристики</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBuilds">
    <property name="title">
     <string>Збірка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBases">
    <property name="title">
     <string>Бази даних компонентів</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSDK">
    <property name="title">
     <string>Менеджер баз даних</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Довідка</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCollections">
    <property name="title">
     <string>Колекції збірок</string>
    </property>
   </widget>
   <addaction name="menuCollections"/>
   <addaction name="menuBuilds"/>
   <addaction name="menuBases"/>
   <addaction name="menuSDK"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Components</class>
   <extends>QWidget</extends>
   <header location="global">components.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Specifications</class>
   <extends>QWidget</extends>
   <header location="global">specifications.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
